### PR TITLE
Fixes for Downloads page: newlines break the links

### PR DESCRIPTION
**Pull request by lluiscampos**
See @eysteins comments in [this already merged PR](https://github.com/mendersoftware/mender-docs/pull/705).

### CODE REVIEW

**Comment by lluiscampos** [comment]
Now I learned how to generate the docs site locally. Great!

The changes look good now. I am merging.

File: 08.Downloads/docs.md [hunk 1]
```diff
@@ -17,14 +17,12 @@ according to your host platform:
 
 Remember to add execute permission (e.g. with `chmod +x mender-artifact`).
 
-!!! If you need to build `mender-artifact` from source, please see [Compiling
-mender-artifact](#compiling-mender-artifact).
+!!! If you need to build `mender-artifact` from source, please see [Compiling mender-artifact](#compiling-mender-artifact).
```
Comment (michaelclelland) [inline 08.Downloads/docs.md]: This link doesn't lead to anywhere, and the text 'compiling mender-artifact' is not within the blue section.

Comment (lluiscampos) [reply 08.Downloads/docs.md]: Now it is within the blue section (I force-pushed), but the link is still broken :( Thanks

**Comment by lluiscampos** [comment]
@michaelatmender Now it should be fine. Can look at it one more time?

**Pull request merged**